site/docs/guides/spark-s3.md

```diff
@@ -0,0 +1,91 @@
+# Accessing data in S3 with Spark
```

Comment:
Overall good document. Thanks for adding.

May be we should add a link to this in `https://projectnessie.org/tools/iceberg/spark/` . Right now it kind of looks hidden in guides section.